backend/src/Squidex/Config/Authentication/LoggingHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Squidex.Config.Authentication
+{
+    public class LoggingHandler : DelegatingHandler
```

Comment:
What is this class? I do not understand the purpose.

Reply:
Sorry. This is a debug class. I'll remove it.